.github/workflows/lint.yml

```diff
@@ -27,3 +27,32 @@ jobs:
     - uses: actions/checkout@v2
     - name: Run shellcheck
       uses: ludeeus/action-shellcheck@1.1.0
+
+  build:
```

Comment:
Måske workflowet så skal hedder "Lint and build" eller lignende?

Reply:
Nah, det ville være "teknisk korrekt", men jeg syntes at navngivningen bør følge intentionen, ikke så meget hvad der i praksis sker. Her "linter" vi, "build" foregår i den anden workflow. Humlen i et build er jo outputtet, som vi i det her tilfælde smider væk.

Reply:
Good point